smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -290,6 +291,33 @@ void restTemplateClient() {
                             .hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping")));
   }
 
+  @Test
+  @org.junit.jupiter.api.Order(3)
```

Comment:
in my opinion you shouldn't enforce the ordering unless it is necessary and when it is it would be nice if there was a comment that explains why you need to do this somewhere in the test class

Reply:
good point - added an explanation at the top

Reply:
would it make sense to only set the oder for the method you need to run before everything else and leave the others to use the default order?

Reply:
good idea